src/TestData/Framework/Provider/RandomGateway.php

```diff
@@ -8,7 +8,7 @@ class RandomGateway extends RandomProvider {
 	protected $gateways = [
 		'paypal-commerce',
 		'stripe',
-		'manual',
```

Comment:
`manual` needs to be kept, which is actually the "gateway" for test donations.

`manual` and `manual_donation` are different.

Reply:
Resolved 5725579e399d036e039cc968e0772ef58da89d9d